mediapipe/tasks/ios/test/vision/image_segmenter/MPPImageSegmenterTests.mm

```diff
@@ -468,6 +468,45 @@ - (void)imageSegmenter:(MPPImageSegmenter *)imageSegmenter
   }
 }
 
+#pragma mark Mask No Copy Tests
+
+- (void)testSegmentWithNoCopyConfidenceMasksAndImageModeSucceeds {
+  MPPImageSegmenterOptions *options =
+      [self imageSegmenterOptionsWithModelFileInfo:kImageSegmenterModelFileInfo];
+
+  MPPImageSegmenter *imageSegmenter = [self createImageSegmenterWithOptionsSucceeds:options];
+  
+  MPPImage *image = [MPPImage imageWithFileInfo:kCatImageFileInfo];
+  [imageSegmenter segmentImage:image withCompletionHandler:^(MPPImageSegmenterResult *result, NSError *error) {
```

Comment:
Don't these test need expectation to ensure that these async blocks execute?

Reply:
No. Because the method returns only after the completion handler call completes and hence the test gets completed only after the result is returned via the handler. This segment method is synchronous as opposed to the live stream segment method.

Reply:
Okay, that's right. I will add a comment so future me does not forget again.